packages/web-react/src/components/Accordion/AccordionContent.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Collapse } from '../Collapse';
+import { AccordionContentProps } from '../../types';
+import { useAccordionStyleProps } from './useAccordionStyleProps';
+import { useAccordionContext } from './AccordionContext';
+import { useAccordionItemContext } from './AccordionItemContext';
+import { useAccordionAriaProps } from './useAccordionAriaProps';
+
+const AccordionContent: React.FC<AccordionContentProps> = ({ children }) => {
+  const { classProps } = useAccordionStyleProps();
+  const { isItemOpen, open } = useAccordionContext();
+  const { id } = useAccordionItemContext();
+
+  const isOpen = React.useMemo(
```

Comment:
I think this function is the same in the Content and Header component. What about creating a hook `useIsAccordionItem`? This hook can access the open variable directly.